rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -244,7 +239,12 @@ object Reduce {
       val starts = jobs.map(_.size).scanLeft(0)(_ + _).toVector
 
       jobs.zipWithIndex
-        .map { case (job, idx) => job.run(starts, idx)(env, rand, costAccountingAlg) }
+        .map {
+          case (job, idx) => {
+            val splitter = split(starts.last, starts(idx))(_, _)
```

Comment:
If you already know `rand` here, why not make `splitter` of type `Int => Blake2b512Random` ?

Reply:
good point